net/mod.ts

```diff
@@ -3,6 +3,15 @@
 /**
  * Network utilities.
  *
+ * ```ts no-assert no-eval
+ * import { getNetworkAddress, getAvailablePort } from "@std/net";
+ *
+ * const port = getAvailablePort();
+ * const hostname = getNetworkAddress();
+ *
+ * Deno.serve({ port, hostname }, () => new Response("Hello, world!"));
```

Comment:
This usage of `hostname` feels a bit misleading to me. Do we recommend using `getNetworkAddress()` in this way?

Reply:
Good point. Fixed.